tests/templates/test_subroutines/test_grover.py

```diff
@@ -50,65 +58,63 @@ def test_id():
     assert op.id == "hello"
 
 
-class TestDecomposition:
-    def test_expand(self):
-        """Tests correct expansion"""
-        wires = tuple(range(3))
+decomp_3wires = [
+    qml.Hadamard,
+    qml.Hadamard,
+    qml.PauliZ,
+    qml.MultiControlledX,
+    qml.PauliZ,
+    qml.Hadamard,
+    qml.Hadamard,
+]
 
-        op = qml.templates.GroverOperator(wires=wires)
 
-        ops = op.expand().operations
+def decomposition_wires(wires):
+    wire_order = [
+        (wires[0],),
```

Comment:
Are the `( ,)` necessary here? Instead could maybe just have `wire_order = [wires[0], ...]` for clarity, and in the check convert using `Wires` to get a proper wires object.

Reply:
`qml.wires.Wires(wires[0]) != wires[0]`, but `qml.wires.Wires(wires[0]) == (wires[0],  )`.

Maybe we should just improve the `__eq__` function for the wires class.

Reply:
oh I get what you mean.  i'll do that.